bin/test_validator

```diff
@@ -15,13 +15,11 @@ shift
 
 site_path=sites/udmi_site_model
 device_id=AHU-1
+proxy_id=AHU-22
 mkdir -p out
 
-serial_no=validator-$RANDOM
-echo Using pubber with serial $serial_no
-
-PUBBER_OUT=pubber.out
-VALIDATOR_OUT=validator.out
+PUBBER_OUT=out/pubber.out
+VALIDATOR_OUT=out/validator.out
```

Comment:
Eventually I wonder if we won't be able to use static filenames and need to do something more robust. e.g.

VALIDATOR_OUT=$(mktemp -p out/ validator.XXXX.out)

Reply:
I often find that actually makes it more difficult overall, because then when you (or a script) wants to automatically process the output you have to figure out which one-of-many files you're dealing with. There's also constraints about multiple-things running at the same time (as in, it won't work because of the UDMI-REFLECTOR registry behavior) -- so practically speaking it shouldn't be an issue in this case.